charts/datadog/templates/kube-state-metrics-core-rbac.yaml

```diff
@@ -91,13 +91,15 @@ rules:
   verbs:
   - list
   - watch
+{{- if .Values.datadog.kubeStateMetricsCore.collectCrdMetrics }}
```

Comment:
https://github.com/DataDog/helm-charts/pull/999 added this RBAC. 
Only when `.Values.datadog.kubeStateMetricsCore.collectCrdMetrics` is enabled, this RBAC needs.

Reply:
To avoid the breaking change, this change should be removed.